modules/update-service-mirror-release.adoc

```diff
@@ -44,7 +44,7 @@ $ OCP_RELEASE=<release_version>
 ----
 +
 For `<release_version>`, specify the tag that corresponds to the version of {product-title} to
-install, such as `4.6.4`.
+install, such as `{product-version}.0`.
```

Comment:
Uhh, I have no idea why [the preview renders this][1] as 4.5.4:

![image](https://user-images.githubusercontent.com/209920/129967589-f6342ec8-2f3f-4527-a3f9-25475f8d7d24.png)

[1]: https://deploy-preview-35601--osdocs.netlify.app/openshift-enterprise/latest/installing/installing-mirroring-installation-images.html#installation-mirror-repository_installing-mirroring-installation-images